A32NX/html_ui/Pages/A32NX_Core/A32NX_Core.js

```diff
@@ -1,29 +1,97 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const ENABLE_TOTAL_UPDATE_TIME_TRACING = false;
+
 class A32NX_Core {
     constructor() {
         this.modules = [
-            new A32NX_ADIRS(),
-            new A32NX_APU(),
-            new A32NX_BaroSelector(),
-            new A32NX_BrakeTemp(),
-            new A32NX_Electricity(),
-            new A32NX_LocalVarUpdater(),
-            new A32NX_FADEC(1),
-            new A32NX_FADEC(2),
-            new A32NX_FWC(),
-            new A32NX_GPWS(this),
-            new A32NX_GroundReference(),
-            new A32NX_Speeds()
+            {
+                name: 'ADIRS',
+                module: new A32NX_ADIRS(),
+                updateInterval: 100,
+            },
+            {
+                name: 'BaroSelector',
+                module: new A32NX_BaroSelector(),
+                updateInterval: 300,
+            },
+            {
+                name: 'BrakeTemp',
+                module: new A32NX_BrakeTemp(),
+                updateInterval: 150,
+            },
+            {
+                name: 'Electricity',
+                module: new A32NX_Electricity(),
+                updateInterval: 100,
+            },
+            {
+                name: 'LocalVars',
+                module: new A32NX_LocalVarUpdater(),
+                updateInterval: 50,
+            },
+            {
+                name: 'FADEC #1',
+                module: new A32NX_FADEC(1),
+                updateInterval: 100,
+            },
+            {
+                name: 'FADEC #2',
+                module: new A32NX_FADEC(2),
+                updateInterval: 100,
+            },
+            {
+                name: 'FWC',
+                module: new A32NX_FWC(2),
+                updateInterval: 50,
+            },
+            {
+                name: 'GPWS',
+                module: new A32NX_GPWS(this),
+                updateInterval: 75,
+            },
+            {
+                name: 'GroundReference',
+                module: new A32NX_GroundReference(),
+                updateInterval: 50,
+            },
+            {
+                name: 'Speeds',
+                module: new A32NX_Speeds(),
+                updateInterval: 200,
```

Comment:
current update interval is 500ms

If you add this feature, please also go ahead remove interval (move the code to the update routine)

Reply:
Fixed